<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomDialog</class>
 <widget class="QWidget" name="RandomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>570</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>751</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>731</width>
     <height>531</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="vertexLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="vertexPushButton">
        <property name="styleSheet">
         <string notr="true">color: rgb(32, 74, 135);
border-color: rgb(32, 74, 135);</string>
        </property>
        <property name="text">
         <string>Generate random vertices</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="edgeLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="edgePushButton">
        <property name="styleSheet">
         <string notr="true">color: rgb(32, 74, 135);
border-color: rgb(32, 74, 135);</string>
        </property>
        <property name="text">
         <string>Generate random edges   </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);
border-color: rgb(32, 74, 135);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>170</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="edgesCountLable">
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 74, 135);
border-color: rgb(32, 74, 135);</string>
      </property>
      <property name="text">
       <string>Edges   Count :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edgeCountLineEdit">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);
border-color: rgb(32, 74, 135);</string>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>170</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="verticesCountLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(32, 74, 135);
border-color: rgb(32, 74, 135);</string>
      </property>
      <property name="text">
       <string>Vertices Count :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="vertexCountLineEdit">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(0, 0, 0);
border-color: rgb(32, 74, 135);
</string>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
